lib/blpapi-wrapper.ts

```diff
@@ -470,7 +471,15 @@ export class Session extends events.EventEmitter {
             if (undefined === cid) {
                 throw new Error('Subscription not found at index ' + i);
             } else {
-                cids.push(_.parseInt(cid));
+                var id: number = _.parseInt(cid);
+                cids.push(id);
+                // unlisten for events
+                _.forEach(SERVICE_TO_SUBSCRIPTION_EVENTS_MAP[
+                              getServiceForSecurity(this.subscriptions[cid].security)
```

Comment:
nit: indent is a bit weird here.  It may be better to capture in a variable what is being passed into `_.forEach`; also it will make this more readable.